terraform/shared/elastic.tf

```diff
@@ -0,0 +1,133 @@
+resource "ec_deployment" "catalogue_api" {
+  name = "catalogue-api"
+
+  region                 = "eu-west-1"
+  version                = "7.13.0"
```

Comment:
much fanciness!

Reply:
as a point, i think this is OK - but it's worth going over the changelog as we have the most complicated use of elastic here, and could break as we're on 7.11 in prod (which we should change).

Reply:
yarr, we'll see when I set up a following index!